conf/pipeline/sarek/munin.config

```diff
@@ -0,0 +1,24 @@
+// Profile config names for nf-core/configs
+
+params {
+  // Specific nf-core/configs params
+  config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
+  config_profile_description = 'nf-core/sarek MUNIN profile provided by nf-core/configs'
+
+  // Specific nf-core/sarek params
+  annotation_cache = true
+  pon = '/data1/PON/vcfs/BTB.PON.vcf.gz'
+  pon_index = '/data1/PON/vcfs/BTB.PON.vcf.gz.tbi'
+  snpEff_cache = '/data1/cache/snpEff/'
+  vep_cache = '/data1/cache/VEP/'
+}
+
+// Specific nf-core/sarek process configuration
+process {
+  withLabel:sentieon {
+    if (params.sentieon) {
```

Comment:
Do you need to define a default value for `params.sentieon` above? Or is this defined in the main pipeline?

Reply:
So `params.sentieon` is set to false on Sarek.
I do think it's safer to add `--sentieon` to the command line, but I'm guessing that it could change at some point.